services.yaml

```diff
@@ -909,49 +926,197 @@ paths:
           description: NOT FOUND
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/folders:
```

Comment:
why not view-folders? we should pick one name when we address these

Reply:
other options i can think of:
Categories, Groups, Sections

Reply:
and the winner is 'categories'! Folders was replaced with categories.